react-debugging.html

```diff
@@ -6,6 +6,25 @@
 
 <!-- HINT: Check the console in the browser developer tools for errors. The lines numbers may not match up with the line numbers in your text editor. -->
 
+
+
+<!--
```

Comment:
really appreciate this breakdown....this step right will set you out amongst your peers. Always find a way to make yourself productively, tactfully rememberable.